src/perform_step/bdf_perform_step.jl

```diff
@@ -16,41 +16,48 @@ end
   alg = unwrap_alg(integrator, true)
   dtₙ, uₙ, uₙ₋₁, uₙ₋₂ = integrator.dt, integrator.u, integrator.uprev, integrator.uprev2
 
+  # TODO: this doesn't look right
```

Comment:
Maybe a later optimization, but we should actually write the Euler step instead of using `perform_step!` for ImplicitEuler. Any change in that function could lead to breakage

Reply:
`ImplicitEuler` most likely won't change in such a way that breaks `ABDF2`. The comment refers to the discontinuity handling.